usaspending_api/search/v2/views/spending_by_category_views/spending_by_agency_types.py

```diff
@@ -35,7 +35,7 @@ def build_elasticsearch_result(self, response: dict) -> List[dict]:
 
             results.append(
                 {
-                    "amount": Decimal(bucket.get("sum_field", {"value": 0})["value"]) / Decimal("100"),
+                    "amount": int(bucket.get("sum_field", {"value": 0})["value"]) / Decimal("100"),
```

Comment:
What makes int more accurate than Decimal?

Reply:
Ran into an issue when testing locally where even though we store values as a scaled_float and multiple by 100 as we sum a value returned from Elasticsearch still had issues around floating point precision. The integer was correct but had an annoying .000001 added on to it. The solution was to truncate at the decimal.

Reply:
OK, tracking now